### PR TITLE
Resolve Front page design issues.

### PR DESCRIPTION
**Pull request by dinesh-sardiwal**
Rerf #10121

first section block remove padding and animations correct.

### CODE REVIEW

File: docroot/sites/all/themes/parallax/sass/front.scss [hunk 1]
```diff
@@ -166,36 +172,31 @@
...
-
     .icon-block {
       opacity: 1;
       @include transform(translateX(0));
-      padding: 17px;
-      margin-top: 8px;
       color: $white;
     }
-  }
+  }
```
Comment (akashjain132) [inline docroot/sites/all/themes/parallax/sass/front.scss]: Extra space is not allowed.

File: docroot/sites/all/themes/parallax/sass/front.scss [hunk 2]
```diff
@@ -166,36 +172,31 @@
   .column.c-left {
     width: 33.3%;
     vertical-align: top;
-    padding: 4%;
+    height:800px;
```
Comment (akashjain132) [inline docroot/sites/all/themes/parallax/sass/front.scss]: You mention fixed height, are you sure about this if any user see in larger screen then may be it breaks

File: docroot/sites/all/themes/parallax/sass/front.scss [hunk 3]
```diff
@@ -152,12 +154,16 @@
   }
 
   .social-buttons {
+  margin-bottom:12%;
```
Comment (akashjain132) [inline docroot/sites/all/themes/parallax/sass/front.scss]: After colon, there must be one character space and also fix indentation

File: docroot/sites/all/themes/parallax/sass/front.scss (hunk 1 above)
Comment (dinesh-sardiwal) [inline docroot/sites/all/themes/parallax/sass/front.scss]: ok i removed it.

File: docroot/sites/all/themes/parallax/sass/front.scss (hunk 3 above)
Comment (dinesh-sardiwal) [inline docroot/sites/all/themes/parallax/sass/front.scss]: Sorry , i have fixed.

File: docroot/sites/all/themes/parallax/sass/front.scss (hunk 2 above)
Comment (dinesh-sardiwal) [inline docroot/sites/all/themes/parallax/sass/front.scss]: Yes akash are you right i have fixed height is auto.

**Pull request merged**